lore/learningobjects/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Course(models.Model):
+    repository = models.ForeignKey(Repository)
+    org = models.TextField()
+    course_number = models.IntegerField()
+    semester = models.TextField()
+    import_date = models.DateField()
+    imported_by = models.TextField()
+
+class LearningObject(models.Model):
+    course = models.ForeignKey(Course)
+    learning_object_type = models.ForeignKey(LearningObjectType)
+    uuid = models.TextField()
+    title = models.TextField()
+    description = models.TextField()
+    content_xml = models.TextField()
+    path_xml = models.TextField()
+    mpath = models.TextField()
+    url_path = models.TextField()
+    parent_id = models.ForeignKey("self")
+    copyright = models.TextField()
+    xa_nr_views = models.TextField()
+    xa_nr_attempts = models.IntegerField()
+    xa_avg_grade = models.FloatField()
+    xa_histogram_grade = models.FloatField()
+
+class LearningObjectType(models.Model):
+    learning_object = models.ForeignKey(LearningObject)
```

Comment:
This foreign key should be removed since there's no reason for learning object types to have an id to a particular learning object